### ISSUE TITLE
Weird messaging in Node 8.x

### ISSUE THREAD

**Issue by stevenvachon** [istanbuljs/nyc#587]
```
Warning: process.on(SIGPROF) is reserved while debugging
```

when doing
```shell
nyc mocha test.js
```

Once I remove the prefixed `nyc`, the warning disappears.

**Comment by bcoe** [comment]
@stevenvachon this should be temporarily fixed in `11.0.1` -- will try to release a permanent fix soon.

**Comment by stevenvachon** [comment]
Thanks!!
